iroha/src/lib.rs

```diff
@@ -21,24 +21,25 @@ use crate::{
     peer::{Peer, PeerId},
     prelude::*,
     queue::Queue,
-    sumeragi::{Message, Sumeragi},
-    torii::Torii,
+    sumeragi::{Message as SumeragiMessage, Sumeragi},
```

Comment:
I think `sumeragi::Message` is enough descriptive, why do wee need `SumeragiMessage`?